test/example_packages/makefile_complex/Makefile

```diff
@@ -0,0 +1,9 @@
+INCLUDE_FLAGS = $(addprefix -I,$(INCLUDE_DIRS))
+
+$(BUILD_DIR)/libmakefile_complex.a: $(BUILD_DIR)/wrapper_mod.o
```

Comment:
I think we need to prefix all the environment variables with `FPM_`, so:

* `FPM_INCLUDE_FLAGS`
* `FPM_BUILD_DIR`

etc.

Reply:
The ones I used are actually the standard environment variables for Makefiles. You (almost) can use an empty Makefile and these would actually work. We can use prefixed variables, but it's one extra difference from what a lot of people will already have and be used to.

Reply:
This seems fine to me - `FC` and `FFLAGS` are standard Makefile variables - and I think we should be minimising the amount of changes people have to do to their Makefiles for compatibility.